src/components/Profile/Profile.tsx

```diff
@@ -0,0 +1,153 @@
+import { Avatar, Box, Button } from '@mui/material';
+
+import { Place, Facebook, Layers, StarRate, People, Person } from '@mui/icons-material';
+
+import { styled } from '@mui/material/styles';
+import Badge from '@mui/material/Badge';
+
+// import { useSelector } from 'react-redux';
```

Comment:
You don't need to leave in commented out code if you have git to do the "remembering" for you.